playbooks/appsemblerPlaybooks/monitoring.yml

```diff
@@ -17,12 +17,12 @@
     - role: "{{ appsembler_roles }}/logstash"
       when: "LOGSTASH_ENABLED|default(False)"
 
-    - role: "{{ appsembler_roles }}/stackdriver"
-      when: "cloud_provider == 'gcp'"
+#    - role: "{{ appsembler_roles }}/stackdriver"
+#      when: "cloud_provider == 'gcp'"
```

Comment:
Why are we disabling stackdriver here?

Reply:
@tkeemon the stackdriver version has been deprecated. Please take a look at this issue: https://github.com/appsembler/roles/issues/50